src/app/ApplicationTemplate.Mobile/iOS/LinkerExclusions.xml

```diff
@@ -1,10 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <linker>
-	<assembly fullname="System">
-		<type fullname="System.Diagnostics.Debug">
-			<method name="WriteLine" />
-		</type>
-	</assembly>
+	<assembly fullname="System" />
```

Comment:
So you used to exclude only `System.Diagnostics.Debug.WriteLine` and now the whole System assembly in excluded? That seems like a big change.

Reply:
What happens if you exclude only System.Diagnostics ?

Reply:
It looks like that in the output.
``` text
1>C:\Program Files\dotnet\packs\Microsoft.iOS.Windows.Sdk\16.1.228\tools\msbuild\iOS\Xamarin.iOS.Common.After.targets(332,3): warning IL2025: Duplicate preserve of 'System.Diagnostics.Debug.WriteLine(Object, String)'.
1>C:\Program Files\dotnet\packs\Microsoft.iOS.Windows.Sdk\16.1.228\tools\msbuild\iOS\Xamarin.iOS.Common.After.targets(332,3): warning IL2025: Duplicate preserve of 'System.Diagnostics.Debug.WriteLine(String)'.
1>C:\Program Files\dotnet\packs\Microsoft.iOS.Windows.Sdk\16.1.228\tools\msbuild\iOS\Xamarin.iOS.Common.After.targets(332,3): warning IL2025: Duplicate preserve of 'System.Diagnostics.Debug.WriteLine(String, Object[])'.
1>C:\Program Files\dotnet\packs\Microsoft.iOS.Windows.Sdk\16.1.228\tools\msbuild\iOS\Xamarin.iOS.Common.After.targets(332,3): warning IL2025: Duplicate preserve of 'System.Diagnostics.Debug.WriteLine(String, String)'.
1>C:\Program Files\dotnet\packs\Microsoft.iOS.Windows.Sdk\16.1.228\tools\msbuild\iOS\Xamarin.iOS.Common.After.targets(332,3): warning IL2025: Duplicate preserve of 'System.Diagnostics.Debug.WriteLine(Object)'.
```

Reply:
@vincentcastagna Didn't see your message, let me see.

Reply:
@vincentcastagna Here is the output.

``` text
4>C:\Program Files\dotnet\packs\Microsoft.iOS.Windows.Sdk\16.1.228\tools\msbuild\iOS\Xamarin.iOS.Common.After.targets(332,3): warning IL2007: Could not resolve assembly 'System.Diagnostics'.
4>C:\Program Files\dotnet\packs\Microsoft.iOS.Windows.Sdk\16.1.228\tools\msbuild\iOS\Xamarin.iOS.Common.After.targets(332,3): warning IL2007: Could not resolve assembly 'System.Diagnostics'.
```

Reply:
`System.Diagnostics` is likely a namespace, not an assembly.

Reply:
Currently running a full build to see if it makes any difference.

Reply:
Looks like it's still working, I don't know may be we could just remove `System`? Why did we add that in the first place?

Reply:
you can check the commit history

Reply:
Looks like it was in the initial commit made by @MatFillion. May be he could tell us more about it?

Reply:
I think this can be revisited at a later stage. You can create a PBI or task for it.